app/logging.py

```diff
@@ -1,79 +1,63 @@
 import logging.config
 import os
+from logging import NullHandler
 from threading import local
 
 import logstash_formatter
 import watchtower
 from boto3.session import Session
 from gunicorn import glogging
+from yaml import safe_load
 
 OPENSHIFT_ENVIRONMENT_NAME_FILE = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
 DEFAULT_AWS_LOGGING_NAMESPACE = "inventory-dev"
-LOGGER_PREFIX = "inventory."
+DEFAULT_LOGGING_CONFIG_FILE = "logconfig.yaml"
+LOGGER_NAME = "inventory"
 
 threadctx = local()
 
 
 def configure_logging(runtime_environment):
```

Comment:
The _runtime_envirnoment_ argument is no longer use. Let’s remove it from here and from all the calls.

```suggestion
def configure_logging():
```